esmvaltool/references/emmons00jgr.bibtex

```diff
@@ -0,0 +1,3 @@
+@article{emmons00jgr,
+    title = "Emmons et al., J. Geophys. Res., 105, D16, 20497-20538, 2000."
```

Comment:
Same for this.

Reply:
@mattiarighi for references with doi information in config-reference.yml, the bibtex entries are retrieved using doi2bib. However, some references are without doi, like emmons00jgr . It is not possible to split them to bibtex entries because they don't follow the template [Last name] et al., [journal abbr.], doi:[doi], [year]. please see my comment here https://github.com/ESMValGroup/ESMValCore/issues/91#issuecomment-575050776

Reply:
done